torch/csrc/api/include/torch/fft.h

```diff
@@ -5,25 +5,113 @@
 namespace torch {
 namespace fft {
 
-#ifndef DOXYGEN_SHOULD_SKIP_THIS
-namespace detail {
+/// Computes the 1 dimensional fast fourier transform over a given axis
+///
+/// \see https://pytorch.org/docs/master/fft.html#torch.fft.fft
+///
+/// Example:
+/// ```
+/// auto t = torch::randn(128, dtype=kComplexDouble);
+/// torch::fft::fft(t);
+/// ```
+inline Tensor fft(const Tensor& self,
+                  c10::optional<int64_t> n=c10::nullopt,
+                  int64_t axis=-1,
+                  c10::optional<std::string> norm=c10::nullopt) {
+  return torch::fft_fft(self, n, axis, norm);
+}
+
+/// Computes the 1 dimensional inverse fast fourier transform over a given axis
+///
+/// \see https://pytorch.org/docs/master/fft.html#torch.fft.ifft
+///
+/// Example:
+/// ```
+/// auto t = torch::randn(128, dtype=kComplexDouble);
+/// torch::fft::ifft(t);
+/// ```
+inline Tensor ifft(const Tensor& self,
+                  c10::optional<int64_t> n=c10::nullopt,
+                  int64_t axis=-1,
+                  c10::optional<std::string> norm=c10::nullopt) {
+  return torch::fft_ifft(self, n, axis, norm);
+}
+
+/// Computes the 1 dimensional FFT of real input with half-complex output
```

Comment:
What does "half-complex output" mean?

Reply:
Same question for the following docs, too.

Reply:
The compact Hermitian representation is complex but only has half the number of values, hence "half-complex". Maybe you would prefer "one-sided"?

Reply:
One-sided sounds better. "half-complex" makes me think only the real or imaginary parts are present (i.e. only half of each complex value).

Reply:
Changed here and tried to use onesided consistently in the python docs as well.